Ska/engarchive/fetch.py

```diff
@@ -69,6 +70,31 @@
 # Cached version (by content type) of first and last available times in archive
 CONTENT_TIME_RANGES = {}
 
+# Data source. Default to CXC. Set to MAUDE by fetch.from_maude()
+_backend = 'CXC'
```

Comment:
For the backend I would propose using this context manager / singleton state variable:
```
class Backend(object):
    _backends = ('cxc',)
    _allowed = ('cxc', 'maude')

    def __init__(self, *backends):
        self._new_backends = backends

    def __enter__(self):
        self._orig_backends = self.__class__._backends
        self.set(*self._new_backends)

    def __exit__(self, type, value, traceback):
        self.__class__._backends = self._orig_backends

    @classmethod
    def set(cls, *backends):
        if any(backend not in cls._allowed for backend in backends):
            raise ValueError('backends {} not in allowed set {}'
                             .format(backends, cls._allowed))
        cls._backends = backends

    @classmethod
    def get(cls):
        return cls._backends
```
You can see how it works below:
```
print(Backend.get())
# ('cxc',)

# Temporarily set different backend within block
with Backend('maude'):
    print(Backend.get())
    # ('maude',)

print(Backend.get())
# ('cxc',)

# Permanently set backend
Backend.set('cxc', 'maude')
print(Backend.get())
# ('cxc', 'maude')
```